src/Services/OrganizationProfileIconService.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Services;
+
+use Blumilk\Meetup\Core\Models\Utils\Constants;
+use Illuminate\Support\Str;
+
+class OrganizationProfileIconService
+{
+    protected string $path;
+
+    public function getPath(string $label): string
+    {
+        $this->path = Constants::PROFILE_ICON_PATH . $label . ".svg";
+
+        return Str::lower($this->path);
+    }
```

Comment:
What do u think about move that into enum `AvailableProfiles` instead of that service. 
Other option is moved that into model and creating icons dynamically instead of saving them into database.